charts/library/common/templates/spawner/_pvc.tpl

```diff
@@ -102,7 +102,11 @@
 
             {{- $credentials := get $.Values.credentials $volsync.credentials -}}
 
-            {{- $url := $credentials.url | replace "http://" "" | replace "https://" "" -}}
+            {{/* Only amazon needs the https:// trimmed, anything else requires it */}}
+            {{- $url := $credentials.url -}}
+            {{- if hasPrefix "https://s3." $url -}}
+              {{- $url = trimPrefix "https://" $url -}}
+            {{- end -}}
```

Comment:
Not sure if its 100% needed. But this is how they list aws endpoints so lets keep it that way.
ie it would probably still work with https even for aws. but I don't have access to aws to check